packages/kbn-securitysolution-io-ts-utils/src/format_errors/index.ts

```diff
@@ -21,7 +21,9 @@ export const formatErrors = (errors: t.Errors): string[] => {
         .map((entry) => entry.key)
         .join(',');
 
-      const nameContext = error.context.find((entry) => entry.type?.name?.length > 0);
+      const nameContext = error.context.find(
+        (entry) => entry.type != null && entry.type.name != null && entry.type.name.length > 0
```

Comment:
Don't miss this syntax... 😅 Seems like there's an issue with using optional chaining in bazel packages for the time being?

Reply:
Correct, I sent a message to the correct channels to ask them if I'm missing something simple.

Reply:
Btw, they gave me feedback on how to re-introduce and possibly fix this. I just opted for the fast way (for now) but down the road we should fix it for sure here.